### PR TITLE
 Update pl directory with current en content

### PR DESCRIPTION
**Pull request by devnook**
This effectively enables Polish as an official language on we.dev. This will alow to send content to gitlocalize on broader scale.
It's staged at https://web-dev-staging.uc.r.appspot.com/pl/measure/

### CODE REVIEW

**Comment by robdodson** [comment]
Hey @devnook can you take a look at the Percy failures? It seems like it's making some articles disappear and some articles get duplicated on the home page.

**Comment by devnook** [comment]
@robdodson it is addressed in this PR: https://github.com/GoogleChrome/web.dev/pull/3752

**Comment by devnook** [comment]
I'm adding TO NOT MERGE label until PR #3752 gets resolved.

**Comment by devnook** [comment]
PTAL

File: src/site/content/pl/pl.11tydata.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+const livePaths = require('../../_filters/live-paths');
```
Comment (robdodson) [inline src/site/content/pl/pl.11tydata.js:1]: It seems like it'd be good for us to remove the paths data from this file in the future so we don't have to update every language if we change the paths.

File: src/site/content/pl/offline.njk [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+layout: layout
+title: Offline
+description: |
+  Network Offline
+noindex: true
+offline: true
+---
+
+<div class="w-layout-container--large">
```
Comment (robdodson) [inline src/site/content/pl/offline.njk:10]: Should this change to an include that uses the `i18n` frontmatter approach to replace the strings?

File: src/site/content/pl/feed.njk [hunk 3]
```diff
@@ -0,0 +1,48 @@
+---
```
Comment (robdodson) [inline src/site/content/pl/feed.njk:1]: Should this change to using an include for the html?

**Comment by robdodson** [comment]
I'm not sure why the Update translated date action is failing. @devnook you should be able to merge this when you're ready if you think it'll be safe to do so.

**Pull request merged**